cmd/api/main.go

```diff
@@ -64,10 +64,11 @@ type fakeServer struct {
 }
 
 var (
+	locationA   = model.Location{Country: "Germany", Region: "eu-west-1", Area: "eu-west-1a"}
```

Comment:
```suggestion
awsIrelandA = model.Location{Country: "Ireland", Region: "eu-west-1", Area: "eu-west-1a"}
```

Also, We should later add a mapping (eu-west-1 -> Ireland -> EU something like that.) We dont need to add it now.